### ISSUE TITLE
Improve data visualization in the stats section

### ISSUE THREAD

**Issue by vanessalane** [tedpedersen/group-project-uc-berkeley#9]
Rather than displaying raw numbers, it would be great to display data in a pretty way

**Comment by tedpedersen** [comment]
check out: 
https://www.chartjs.org 
https://d3js.org
https://www.highcharts.com/maps/demo

**Comment by tedpedersen** [comment]
Should we add a label here, for 1st time users, may not sure what Stats these are, maybe just "US STATISTICS":
<img width="1240" alt="Screen Shot 2020-07-02 at 2 47 22 PM" src="https://user-images.githubusercontent.com/929079/86412097-1c489080-bc73-11ea-9e3e-dc0706c80fc9.png">

**Comment by tedpedersen** [comment]
I think the search should have the typeahead it made for good UX, but up to you.

**Comment by vanessalane** [comment]
Agreed on typehead! I'm just working section by section and will add it in once I'm there.

**Comment by tedpedersen** [comment]
Awesome! Looking great so far, nice work!!!

**Comment by vanessalane** [comment]
Simple improvements:
- add percentage of global to the existing stats
- ~color text or just arrow red/yellow/green~ (done)
- ~consolidate new/total cases like the google dashboard~  (done)
- pie chart of case status for that country (replace the existing chart) --> not sure this is a great way to represent data, may just look nice.

Some more involved data visualization ideas:
- Table that sorts countries by case load (figure out the best way to represnet the different statuses)
- table that shows case load at the province level for that country
- Line chart that shows total, active, for a given country to show change over time

If adding in data visualization, determine whether to create a separate page for the stats (may make the home page too busy)
